crm_claim_type/views/crm_claim_stage.xml

```diff
@@ -28,10 +28,7 @@
         <field name="arch" type="xml">
             <xpath expr="//field[@name='name']" position="after">
                 <field name="claim_common" />
-                <field
-                    name="claim_type"
-                    attrs="{'invisible':[('claim_common','=',True)]}"
-                />
+                <field name="claim_type" />
```

Comment:
You miss the invisible attribute

Reply:
I deleted it on purpose, it's doing nothing in the view.